### PR TITLE
Adding Dapr quickstart

### PR DESCRIPTION
**Pull request by harrykimpel**
# Summary

Adding some quickstart dashboards for Dapr (www.dapr.io).

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: install/third-party/dapr/install.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+id: third-party-dapr
```
Comment (MichelLosier) [inline install/third-party/dapr/install.yml:1]: Hi @harrykimpel we've been in the process of deprecating install plans in favor of data sources, and quickstarts now require a data source over an install plan.

More information here: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1

File: quickstarts/dapr/config.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+id: 658d36f3-aa96-4e74-93da-b74eefcaabd3
```
Comment (MichelLosier) [inline quickstarts/dapr/config.yml:1]: New quickstarts will have their id generated after their first merge, lets remove this field

File: quickstarts/dapr/config.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - Harry Kimpel
+keywords:
+  - apm
+  - dapr
+dashboards:
+  - dapr
+installPlans:
+  - third-party-daprr
+dataSourceIds:
+  - dapr
```
Comment (MichelLosier) [inline quickstarts/dapr/config.yml:21]: Looks like this data source doesn't exist yet, we'll need to create a new `dapr` data source in this PR as mentioned above.

**Review by MichelLosier — CHANGES_REQUESTED** [review]

File: install/third-party/dapr/install.yml (hunk 1 above)
Comment (harrykimpel) [reply install/third-party/dapr/install.yml:1]: @MichelLosier I kept the install plan for now, let me know if you want me to remove it. I also added a data source.

File: quickstarts/dapr/config.yml (hunk 2 above)
Comment (harrykimpel) [reply quickstarts/dapr/config.yml]: id removed

File: quickstarts/dapr/config.yml (hunk 3 above)
Comment (harrykimpel) [reply quickstarts/dapr/config.yml:20]: added the data source

**Comment by MichelLosier** [comment]
Hey @harrykimpel looking great! Something I missed on the first pass is that it looks like we have two dashboards in the same directory, which our validation is complaining about. Our pipeline currently expects one dashboard defined per dashboard directory, so will need to split those dashboards out into respective directories. [More on dashboards](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards)

**Comment by harrykimpel** [comment]
> More on dashboards

Thank you @MichelLosier. I moved the dashboards into separate folders and updated the quickstarts config to refer to these two dashboards. I hope this is as expected. If not, how do others deal with multiple dashboards. I guess it is not that unusual, right?

**Review by MichelLosier — APPROVED** [review]

**Comment by MichelLosier** [comment]
Looks great @harrykimpel ! And yep this is exactly how you handle multiple dashboards thanks!

**Comment by harrykimpel** [comment]
> Looks great @harrykimpel ! And yep this is exactly how you handle multiple dashboards thanks!

Thank you @MichelLosier! I promise to remember all of this next time ;-)

**Pull request merged**